website/showcase.json

```diff
@@ -165,5 +165,12 @@
       "Blog post": "https://medium.com/vis-gl/rendering-a-minecraft-world-with-deck-gl-927b52c5c6db"
     },
     "description": "Rendering A Minecraft World With deck.gl."
+  },
+  {
```

Comment:
Don't you want it at the top, as it is the latest / newest show case? Anyone coming back to this page probably won't look into the bottom of the list.

Reply:
Yes I can change it to be first if no one minds.